CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ endif()
 # and add set(OPTION VALUE) to override these build options.
 # Alernatively, use cmake -DOPTION=VALUE through command-line.
 dgl_option(USE_CUDA "Build with CUDA" OFF)
-dgl_option(USE_OPENMP "Build with OpenMP" ON)
+dgl_option(USE_OPENMP "Build with OpenMP" OFF)
```

Comment:
Should we change this? I suppose we just need to customize this in `config.cmake`